content/releases.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Releases
+layout: page
+
+---
+
+# Releases
+
+This page describes the release process as well as the currently planned release schedule for upcoming releases as well as the respective release schepherds. Release shepherds are chosen on a voluntary basis.
```

Comment:
This sentence has "as well" two times and repeats "release" one too many times IMO :)

How about "This page describes the release process, the currently planned schedule for upcoming releases, as well as ..."